urllib3/response.py

```diff
@@ -387,48 +389,63 @@ def readinto(self, b):
             b[:len(temp)] = temp
             return len(temp)
 
-    def read_chunked(self, amt=None):
+    def _update_chunk_length(self):
+        # First, we'll figure out length of a chunk and then
+        # we'll try to read it from socket.
+        if self.chunk_left is None:
+            line = self._fp.fp.readline()
+            line = CHUNKED_SPLIT_RE.split(line, 1)[0]
```

Comment:
For some reason, I was under the impression that bytes objects didn't support `.split` but apparently I'm wrong:

```py
~ python3.4
Python 3.4.2 (default, Oct  8 2014, 19:29:52)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.51)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> b'foo; bar'.split(b';')
[b'foo', b' bar']
```

Might have been a problem with 3.2 or 3.3 but I'm probably just wrong.